app/helpers/alert_notifier_helper.rb

```diff
@@ -60,4 +60,16 @@ def new_subscription_url_with_tracking(alert: nil, utm_content: nil)
 
     new_subscription_url(params)
   end
+
+  def subject(alert, applications, comments)
+    items = if !applications.empty? && comments.empty?
```

Comment:
I think this might be a bit clearer if you use the Enumerable method .any? instead of ![stuff].empty . I found this quite slow to parse what each line was asking.

Also, it could help to put the conditionals indented under the variable assignment to show that it's all inside that, [like this example](https://github.com/bbatsov/ruby-style-guide#use-if-case-returns).